AppBidding-MoPub/AppBidding-MoPub/AdConfigurations.h

```diff
@@ -0,0 +1,28 @@
+//
+//  AdConfigurations.h
+//  AppBidding-MoPub
+//
+//  Copyright © 2020 Criteo. All rights reserved.
```

Comment:
The LICENSE file in this project indicates how to apply the copyright: https://github.com/criteo/ios-publisher-sdk-examples/blob/master/LICENSE#L180

The following text should be used:
```
   Copyright 2020 Criteo

   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
```

Reply:
License headers updated